cobc/codegen.c

```diff
@@ -7115,18 +7117,103 @@ output_java_call (struct cb_call *p)
 	method_name = last_dot + 1;
 	class_name = class_and_method_name;
 
+    for(ptr = p->args; ptr != NULL; ptr = ptr->next) {
```

Comment:
Note: indentation is strange here. I think you need to adjust that in your settings for this project.